kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/core/v1/PodOperationsImpl.java

```diff
@@ -468,33 +470,76 @@ private void copyFile(String source, File target)  {
         String filename = parts[parts.length - 1];
         destination = destination.toPath().resolve(filename).toFile();
     }
-    try (InputStream is = readFile(source);) {
-      Files.copy(is, destination.toPath(), StandardCopyOption.REPLACE_EXISTING);
+
+    try {
+      Files.deleteIfExists(destination.toPath());
```

Comment:
Won't the FileOutputStream default to overwriting?

Reply:
Right. Fixed.